readme.md

```diff
@@ -56,10 +56,17 @@ OpenRCT2 requires original files of RollerCoaster Tycoon 2 to play. It can be bo
 - [7-Zip](http://www.7-zip.org/) (for deployment only)
 - [NSIS](http://nsis.sourceforge.net/) (for deployment only)
 
-### Mac:
+### macOS:
+
+- Xcode 8
+
+The program can also be built as a command line program using CMake. This type of build requires:
```

Comment:
This is the 'Console' application.
![image](https://cloud.githubusercontent.com/assets/737603/19518604/5cc99130-9608-11e6-9397-dc9d2847c9ff.png)

Xcode calls it a 'Command Line Tool', so that term will probably the standard way to call it.
![image](https://cloud.githubusercontent.com/assets/737603/19518742/0706a1f6-9609-11e6-9af2-7507122f9c2f.png)